### PR TITLE
Add try from str

### PR DESCRIPTION
**Pull request by jamesmunns**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [package]
 name = "base_emoji"
-version = "1.0.0"
+version = "1.0.1"
```
Comment (badboy) [inline Cargo.toml]: Should be 1.1.0, but just remove this part, then I use [`cargo-releaes`](https://github.com/sunng87/cargo-release) to make it happen

**Review by badboy — COMMENTED** [review]
Only the versioning thing, but then it can go in

**Review by badboy — APPROVED** [review]

**Comment by jamesmunns** [comment]
@badboy rebased away the version change commit, should be good to go now!

**Review by badboy — APPROVED** [review]

**Comment by jamesmunns** [comment]
@badboy okay to merge and release?

**Pull request merged**